docs/quick-start.md

```diff
@@ -0,0 +1,92 @@
+	- [You can read more about using the CLI here](https://github.com/Automattic/jetpack/blob/master/tools/cli/README.md).
+
+## Running Jetpack locally
+
+After everything is installed, you're ready to run Jetpack locally! While there are other supported methods of doing this, we recommend and support using Docker containers. 
+
+To setup Docker:
+- Run `brew install --cask docker` or [download the latest version here](https://www.docker.com/products/docker-desktop). 
+- Copy the settings file so you can modify passwords and other configurations: `cp tools/docker/default.env tools/docker/.env`
+- Start the Docker container using `jetpack docker up`
+- Install WordPress in your Docker container using `jetpack docker install` 
+- Open up http://localhost to see your site!
+
+For more in depth docker instructions, follow the [Docker environment for Jetpack Development guide](../tools/docker/README.md).
+
+### Setting up Jurassic Tube
+
+To test network related features of Jetpack, you'll need a test site that can create local HTTP tunnels. If you're an Automattician, we recommend using Jurassic Tube:
+
+- Add a subdomain on [jurassic.tube](https://jurassic.tube/)
+- Make sure Docker is running `jetpack docker up -d`
+- Make sure you've run `pnpm install && pnpm cli-link`
+- Stand on the monorepo root in your terminal and run `mkdir tools/docker/bin/jt`
+- Run the installation script: `chmod +x tools/docker/bin/jt/installer.sh && tools/docker/bin/jt/installer.sh`
+- Use `jetpack docker jt-up your-username custom-subdomain` to start the site.
+- Your site should be avalable at `https://custom-subdomain.jurassic.tube`
+
+*Optional*
+
+To make things easier, you can configure Jurassic Tube to remember your username and subdomain.
+
+Let’s assume somebody with WP.com username david uses the URL https://spaceman.jurassic.tube:
+
+- Set your username: `jetpack docker jt-config username [your-username-here e.g david]`
+- Set your subdomain: `jetpack docker jt-config subdomain [your-subdomain-here e.g. spaceman]` 
+- Now, you can start your site with `jetpack docker jt-up`
+
+Note: This is for Automattician use only. For other methods, check out [ngrok](https://github.com/Automattic/jetpack/blob/master/tools/docker/README.md#using-ngrok-with-jetpack) or [another similar service](https://alternativeto.net/software/ngrok/).
+
+### Check if your environment is ready for Jetpack development
+
+We provide a script to help you in assessing if everything's ready on your system to contribute to Jetpack.
+
+```sh
+tools/check-development-environment.sh
+```
+
+Running the script will tell you if you have your environment already set up and what you need to do in order to get it ready for Jetpack development.
+
+If you're ready to start, you should see all green `SUCCESS` messages. If the script detect issues, you will see a a red `FAILED` note and a link that will help you figure out what you need to change/fix to address the issue.
+
+# Development workflow
```

Comment:
I think part of setting it up should be getting the unit tests running. phpunit, etc.

Once a dev knows where the tests are they can then iterate a lot more independently. Linters and stuff would be good to.

Even just listing the commands is a good step with some links to where the main docs for those tests/linters/etc are. Don't need to document them here.

Reply:
> Usable version of PHP ... ok (version 8.1.0)

I don't think it is intentional, but 8.1.0 is a bit too new between WP still throwing a lot of deprecations and WordPress CS rules of PHPCS straight fails under 8.1.0 in a few cases. (fixed upstream, but not released).

> I think part of setting it up should be getting the unit tests running. phpunit, etc.

We should add how to run those in the guide. Once you have everything setup in the guide (e.g. composer, docker), it's a matter of running `jetpack test` for the project/type of tests and noting that tests exist in each project's directory.

We don't have a single entry point for lining yet so that would be a separate piece of work, but could link to a document (and create it if needed) on how to run the linters directly and when do they run automatically (e.g. upon commit on changed lines already happens, etc).

Reply:
Note the whole message is

> * Usable version of PHP                    ... ok (version 8.1.0)
>  PHP at /usr/bin/php is version 8.1.0. We've only tested with PHP up to 8.0.
>  See https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#php

(with the "ok (version 8.1.0)" bit in yellow, not green)

Reply:
Added a line about using `jetpack test` for now.